Resources/Prototypes/_Shitmed/Research/civilianservices.yml

```
@@ -61,4 +61,4 @@
   tier: 3
   cost: 10000
   recipeUnlocks:
-  - OmnimedTool
\ No newline at end of file
+  - OmnimedTool
```

yamllint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)